MapboxNavigation/NavigationMapView.swift

```diff
@@ -677,12 +670,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
                 source.shape = arrowShape
             } else {
                 arrow.minimumZoomLevel = minimumZoomLevel
-                arrow.lineCap = MGLStyleValue(rawValue: cap)
-                arrow.lineJoin = MGLStyleValue(rawValue: join)
-                arrow.lineWidth = MGLStyleValue(interpolationMode: .exponential,
-                                                cameraStops: routeLineWidthAtZoomLevels.multiplied(by: 0.70),
-                                                options: [.defaultValue : MGLConstantStyleValue<NSNumber>(rawValue: 1.5)])
-                arrow.lineColor = MGLStyleValue(rawValue: .white)
+                arrow.lineCap = NSExpression(forConstantValue: cap)
+                arrow.lineJoin = NSExpression(forConstantValue: join)
+                arrow.lineWidth = NSExpression(format: "FUNCTION($zoomLevel, 'mgl_interpolateWithCurveType:parameters:stops:', 'linear', nil, %@)", argumentArray: [routeLineWidthAtZoomLevels.multiplied(by: 1.5)])
```

Comment:
Since you know in advance how many arguments this format string needs, use `NSExpression(format:)` instead of `NSExpression(format:argumentArray:)`. The first argument to `NSExpression(format:)` takes a comma-separated list of arguments following the format string.